### PR TITLE
[TIMEDATE] Do not display a message box when NTP sync fails

### PR DESCRIPTION
**Pull request by Fraizeraust**
## Purpose
In Windows XP and Server 2003 the Time/Date control applet does not display any message box in case of NTP synchronization failure or no given NTP server name. However, their applet displays a dynamic string resource every time a successful or failed NTP synchronization operation occurs.

For now, this PR merely checks if the string is empty and displaying (only) a debug information to the debugger, at disposal of displaying a message box. I might implement such dynamic resource string to the CPL later in a next PR.

**JIRA Issue:** [CORE-16654](https://jira.reactos.org/browse/CORE-16654)

### CODE REVIEW

**Comment by Fraizeraust** [comment]
Cc @ThFabba / @Doug-Lyons

File: dll/cpl/timedate/internettime.c [hunk 1]
```diff
@@ -278,7 +278,7 @@ InetTimePageProc(HWND hwndDlg,
                     dwError = W32TimeSyncNow(L"localhost", 0, 0);
                     if (dwError != ERROR_SUCCESS)
                     {
-                        DisplayWin32Error(dwError);
+                        DPRINT("Failed to syncrhonize the time! Invalid NTP server name has been caught or no server name could be found. (Error - %lu)\n", dwError);
```
Comment (learn-more) [inline dll/cpl/timedate/internettime.c]:
```suggestion
                        DPRINT("Failed to synchronize the time! Invalid NTP server name has been caught or no server name could be found. (Error - %lu)\n", dwError);
```

File: base/services/w32time/w32time.c [hunk 2]
```diff
@@ -191,6 +191,13 @@ SetTime(VOID)
 
     DPRINT("Time Server is '%S'.\n", szData);
 
+    /* Is the given string empty? */
+    if (szData[0] == '\0')
```
Comment (learn-more) [inline base/services/w32time/w32time.c]:
```suggestion
    if (cbName == 0 || szData[0] == '\0')
```

**Comment by Doug-Lyons** [comment]
This looks good to me. Thanks.

File: dll/cpl/timedate/internettime.c [hunk 3]
```diff
@@ -278,7 +278,7 @@ InetTimePageProc(HWND hwndDlg,
                     dwError = W32TimeSyncNow(L"localhost", 0, 0);
                     if (dwError != ERROR_SUCCESS)
                     {
-                        DisplayWin32Error(dwError);
+                        DPRINT("Failed to synchronize the time! Invalid NTP server name has been caught or no server name could be found. (Error - %lu)\n", dwError);
```
Comment (katahiromz) [inline dll/cpl/timedate/internettime.c]:
```suggestion
                        DPRINT("Failed to synchronize the time! Invalid NTP server name has been caught or no server name could be found. (Error: %lu)\n", dwError);
```

**Pull request merged**